spec/beaker-pe/pe-client-tools/executable_helper_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require 'beaker'
+require 'scooter'
```

Comment:
ahh, good catch @jpartlow . I have a ticket currently in RE's sprint to approve scooter for release to the general public. Is this issue something that we can wait on until that approval is done, or do we need to address this issue sooner?